config/config.py.template

```diff
@@ -340,6 +340,23 @@ STREAMING_HDFS_RECOVERY_TEST_OPTS = STREAMING_COMMON_OPTS + streaming_batch_dura
     OptionSet("file-cleaner-delay", [300])
 ]
 
+STREAMING_RECEIVER_LOAD_BALANCE_TEST_OPTS = streaming_batch_duration_opts(2000) + [
+    OptionSet("total-duration", [30]),
+    OptionSet("hdfs-url", [HDFS_URL]),
+    OptionSet("total-executor-cores", [20], can_scale=True),
```

Comment:
@tdas do you know how to get `total-executor-cores` and `num-executors` in codes?